src/js/components/ServicesBreadcrumb.js

```diff
@@ -74,9 +75,15 @@ class ServicesBreadcrumb extends React.Component {
       );
     }
 
+    const classes = classNames('flex-box control-group', this.props.className);
+    const h4Classes = classNames(
```

Comment:
#nit shouldn't these variable names reflect the props? `className` and `h4ClassNames` – also we should standardize on the plural form. #feelfreetoignore